File: 0062-unique-paths/0062-unique-paths.cs
public class Solution {
    public int UniquePaths(int m, int n) {
        int[,] grid = new int[m,n];

        for (int r = 0; r < m; r++){
            for (int c = 0; c < n; c++){
                grid[r,c] = (r > 0 && c > 0) ? grid[r-1, c] + grid[r, c-1] : 1; 
            }
        }
        return grid[m-1,n-1];
    }
}

File: 0062-unique-paths/0062-unique-paths.java
class Solution {
    public int uniquePaths(int m, int n) {
        int[][] pa = new int[m][n];
        for (int i = 0; i < m; i++){
            for (int j = 0; j < n; j++){
                if (j == 0 || i == 0) pa[i][j] = 1;
                else pa[i][j] = pa[i-1][j] + pa[i][j-1];
            }
        }


        return pa[m-1][n-1];
    }
}